src/Microsoft.Performance.SDK.Tests/ProcessingOptionsResolverTests.cs

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.SDK.Processing;
+using Microsoft.Performance.SDK.Processing.DataSourceGrouping;
+using Microsoft.Performance.SDK.Processing.Options;
+using Microsoft.Performance.SDK.Runtime;
+using Microsoft.Performance.Testing;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source123;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source4;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source5;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Performance.SDK.Tests
+{
+    [TestClass]
+    public class ProcessingOptionsResolverTests
+    {
+        [TestMethod]
+        [IntegrationTest]
```

Comment:
I don't think this is an integration test since it appears to be testing the spec of `GlobalProcessingOptionsResolver` claiming "Each call to GetProcessorOptions returns the given value."